__tests__/app.test.js

```diff
@@ -179,4 +179,46 @@ describe('app',() => {
         })
         
     })
+    describe.only('POST- /api/reviews/:review_id/comments', ()=>{
+        test('201: accepts an object with properties username and body, also responds with the posted comment', ()=>{
+            const newComment = {
+                username: 'dav3rid',
+                body: 'this game made my feet hurt and caused a family argument'
+            };
+            return request(app)
+            .post('/api/reviews/3/comments')
+            .send(newComment)
+            .expect(201)
+            .then(({body})=>{
+                const comment = body.commentInfo;
+                console.log(comment)
+                expect(comment).toHaveLength(1);
+                expect(comment[0]).toMatchObject({
+                    comment_id: expect.any(Number),
+                    body: expect.any(String),
+                    votes: expect.any(Number),
+                    author : expect.any(String),
```

Comment:
You know the author and the body so test exactly for that. Also remove the console.log